server/client.go

```diff
@@ -238,6 +240,15 @@ func (client JiraClient) UpdateComment(issueKey string, comment *jira.Comment) (
 	return updated, err
 }
 
+// GetStatuses returns a list of all statuses associated with JIRA workflows.
+func (client JiraClient) GetStatuses() ([]jira.Status, error) {
```

Comment:
Can you please clarify: are these all statuses for the "instance"? "project"? it's not obvious how this request is contextualized. Maybe an API doc link?